api/api/spec/spec.yaml

```diff
@@ -11532,6 +11552,8 @@ paths:
           $ref: '#/components/responses/UnauthorizedResponse'
         '403':
           $ref: '#/components/responses/PermissionDeniedResponse'
+        '404':
+          $ref: '#/components/responses/InvalidUriResponse'          
```

Comment:
Mmm I see. The `%2F` code is a URL encoded `/`, so, as it was shown in the logs, it would be equivalent to doing:
- `GET /lists/files/te/st`

Endpoints never expect `/` in their path parameters, as they would be considered different (unexistent) endpoints. This same thing would happen with any endpoint actually, not only with those expecting a path parameter:
```JSON
# curl -k -X GET "https://localhost:55000/agents/test" -H "Content-Type:application/json" -H "Authorization: Bearer $TOKEN"
{"title": "Not Found", "detail": "404: Not Found"}
``` 

If instead of including an encoded `/` we include a `-` (`%2D`), it works correctly:
```JSON
# curl -k -X GET "https://localhost:55000/lists/files/te%2Dst?pretty=true" -H "Content-Type:application/json" -H "Authorization: Bearer $TOKEN"
{
   "data": {
      "affected_items": [],
      "total_affected_items": 0,
      "total_failed_items": 1,
      "failed_items": [
         {
            "error": {
               "code": 1802,
               "message": "Lists file not found",
               "remediation": "Please, use `GET /lists/files` to find all available lists"
            },
            "id": [
               "te-st"
            ]
         }
      ]
   },
   "message": "No list was returned",
   "error": 1
}
```

The conclusion is that the behavior of these endpoints is not different from the rest, so we do not need to specify 404 among their possible responses.

Reply:
Understood; I removed error 404 from the endpoints in the `specs.yaml`.